src/openrct2/network/NetworkBase.cpp

```diff
@@ -42,7 +42,7 @@
 // This string specifies which version of network stream current build uses.
 // It is used for making sure only compatible builds get connected, even within
 // single OpenRCT2 version.
-#define NETWORK_STREAM_VERSION "17"
+#define NETWORK_STREAM_VERSION "18"
```

Comment:
why does network stream version need to be changed?

Reply:
Because it will change the outcome of the simulation